src/lib/tvgCommon.h

```diff
@@ -25,6 +25,10 @@
 #include "config.h"
 #include "thorvg.h"
 
+#ifdef _BUILD_WIN_
+#include <cstring>
```

Comment:
I think we can isolate this include to the necessary files.
Could you please suggest the error position?

Reply:
@hermet @JSUYA before I noticed this pr I prepared mine: https://github.com/Samsung/thorvg/pull/840

I dodn;t want to include string.h in tvgInitializer.cpp so I changed the code a bit.
I would appreciate your comment on such an approach.

Reply:
I deleted the code change for the build fix.
I hope that build fail will be fixed in a separate patch(add cstring) or #840.